experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -61,8 +61,19 @@ export function makeField(
 	fieldSchema: FieldStoredSchema,
 	cursor: ITreeSubscriptionCursor,
 ): EditableField {
-	const targetSequence = new FieldProxyTarget(context, fieldSchema, cursor);
-	return adaptWithProxy(targetSequence, fieldProxyHandler);
+	const fieldAnchor = cursor.buildFieldAnchor();
+
+	const targetSequence = new FieldProxyTarget(context, fieldSchema, cursor, fieldAnchor);
```

Comment:
I looked at ProxyTarget, and it seems like simplifying it so the anchor argument is required (and readonly) should be pretty easy now. If you do that you can delete the abstractBuildAnchor method and some ifs so I think it would be a good idea to include that in this change.

Reply:
simplified `ProxyTarget` to require anchor argument